fixture/trial-license-cert.pem

```diff
@@ -0,0 +1,15 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
I just copied from this certificate file from https://scalardb.scalar-labs.com/docs/latest/scalar-licensing/?scalardl-ledger=scalardl-ledger-trial#scalardl-ledger .